### DISCUSSION TITLE
can't login

### DISCUSSION THREAD

**Discussion by gitmengzh** [reworkd/AgentGPT#772] [Q&A]
hi, guess
iI have deployed AgentGPT in Docker on a cloud server. The project started successfully, but I cannot log in as the link redirection failed.
Has anyone encountered this situation before?
![image](https://github.com/reworkd/AgentGPT/assets/13773713/021eab85-2b84-4738-a891-3b0a2751c542)
input any username, click the page 
![image](https://github.com/reworkd/AgentGPT/assets/13773713/47e2fd64-a630-49b5-96c6-03b8c9ed29ed)

**Comment by qingxuantang** [comment]
Encountered the same issue... Haven't figuring out why...

**Reply by gitmengzh** [reply]
> Encoun

try to input "Agentgpt", I have built a new environment in the local , and login successfully, but in the cloud environment, it still does not work

**Reply by qingxuantang** [reply]
As I understand, this issue occurs when the site redirects to localhost:3000 after clicking the sign in button... Since there's no localhost on the server, it doesn't work.  I tried once on my pc's google chrome, with my local agentgpt also running( on localhost), after signing in via the browser, the site redirected to localhost which was actually my local AgentGPT env LOL...

**Reply by gitmengzh** [reply]
yep, this means that it is not possible to log in on the cloud service？

**Comment by vzwjustin** [comment]
> hi, guess
> iI have deployed AgentGPT in Docker on a cloud server. The project started successfully, but I cannot log in as the link redirection failed.
> Has anyone encountered this situation before?
> ![image](https://github.com/reworkd/AgentGPT/assets/13773713/021eab85-2b84-4738-a891-3b0a2751c542)
> input any username, click the page 
> ![image](https://github.com/reworkd/AgentGPT/assets/13773713/47e2fd64-a630-49b5-96c6-03b8c9ed29ed)
> 
> 

I'm having the SAME EXACT PROBLEM!!

**Comment by jasangill1** [comment]
Hello, would you happen to be located in China?

**Reply by vzwjustin** [reply]
I'm in the United States

**Reply by qingxuantang** [reply]
I am in China, yes

**Reply by jasangill1** [reply]
I Believe openai is banned in china this maybe causing your problem

**Comment by clickbrain** [comment]
I believe I am having the same issue. I am using Docker and when the application loads and you try to initiate anything, it asks for a login. Whatever I put in does a redirect to a 404 error on localhost. 

#1219 - I replied on this issue as well. Would love to try this, but can't get past this issue.

**Reply by awtkns** [reply]
Hey @clickbrain would you be able to share some screenshots of the error logs?

**Reply by jasangill1** [reply]
Hello @clickbrain try deleting your docker volume you may have an old DB which is causing this. then run 'docker compose build'

**Reply by clickbrain** [reply]
Will give it a shot. Thanks!
